sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/AggregatePushDownUtils.scala

```diff
@@ -106,7 +107,9 @@ object AggregatePushDownUtils {
       // aggregate push down simple and don't handle this complicate case for now.
       return None
     }
-    aggregation.groupByColumns.foreach { col =>
+    aggregation.groupByExpressions.foreach { expr =>
+      assert(expr.isInstanceOf[FieldReference])
```

Comment:
how can we make this assertion? do you mean the v2 pushdown framework still only support group by attrs?